core/quantity/src/Formatter/Format.ts

```diff
@@ -10,8 +10,8 @@ import { QuantityConstants } from "../Constants";
 import { QuantityError, QuantityStatus } from "../Exception";
 import { UnitProps, UnitsProvider } from "../Interfaces";
 import {
-  DecimalPrecision, FormatTraits, formatTraitsToArray, FormatType, formatTypeToString, FractionalPrecision,
-  getTraitString, parseFormatTrait, parseFormatType, parsePrecision, parseRatioType, parseScientificType, parseShowSignOption,
+  DecimalPrecision, FormatTraits, formatTraitsToArray, FormatType, FractionalPrecision,
+  getTraitString, parseFormatTrait, parseFormatType, formatTypeToString, parsePrecision, parseRatioType, parseScientificType, parseShowSignOption,
```

Comment:
since you aren't using formatTypeToString anymore, can the import statement be removed?